playground/backend/configs/SDK_JAVA.json

```diff
@@ -8,6 +8,7 @@
   ],
   "run_args": [
     "-cp",
-    "bin:"
+    "bin:",
+    "-Djava.util.logging.config.file={logConfigFile}"
   ]
 }
```

Comment:
Please, add an empty line at the end of this file

Reply:
Done.